projects/openapi-cli/src/operations/streams/undocumented.ts

```diff
@@ -74,27 +93,25 @@ export class UndocumentedOperations {
     spec: OpenAPIV3.Document,
     specUpdates?: AsyncIterable<OpenAPIV3.Document>
   ): UndocumentedOperations {
+    const hostBaseMap: { [key: string]: string } = {};
 
-    const basePaths =
-      spec.servers?.map((server) => {
-        // add absolute in case url is relative (valid in OpenAPI, ignored when absolute)
-        const parsed = new Url.URL(server.url, 'https://example.org');
+    spec.servers?.forEach((server) => {
```

Comment:
Robust fix for the basePath logic. The main issues we've been having opened on the repo relate to this. The previous logic was not using the host (mistake) and brittle with trailing slashes.